node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala

```diff
@@ -0,0 +1,132 @@
+package com.wavesplatform.it.sync.grpc
+
+import java.nio.charset.StandardCharsets
+
+import com.google.protobuf.ByteString
+import com.wavesplatform.it.NTPTime
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.sync._
+import com.wavesplatform.it.util._
+import com.wavesplatform.protobuf.transaction.PBTransactions
+import com.wavesplatform.common.utils.EitherExt2
+import com.wavesplatform.lang.ValidationError
+import com.wavesplatform.transaction.TxValidationError.{InvalidName, NonPositiveAmount, TooBigArray}
+import io.grpc.Status.Code
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class IssueTransactionGrpcSuite extends GrpcBaseTransactionSuite with NTPTime with TableDrivenPropertyChecks {
+
+  val (issuer,issuerAddress) = (firstAcc,firstAddress)
+
+  test("asset issue changes issuer's asset balance") {
```

Comment:
add issue nft asset and with decimals 8
also add tests for boundary conditions for asset name